components/ai_chat/resources/page/components/privacy_message/index.tsx

```diff
@@ -19,7 +27,13 @@ function PrivacyMessage () {
       <section>
         <b>{getLocale('aboutTitle')}</b>
         <p>{getLocale('aboutDescription')}</p>
-        <p>{getLocale('aboutDescription_2')}</p>
+        <p>
+          {aboutDescription2.beforeTag}
+          <a onClick={handleAnchorClick} href="https://google.com" target="_blank">
```

Comment:
It feels a bit weird having the placeholder `href` pointing to Google (though I guess we don't have the hover preview or right click support, so I it's probably fine).

Reply:
This was a left-over, nice catch!

Reply:
Changed